pootle/static/js/browser/components/TopContributors.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import Avatar from 'components/Avatar';
+import { t } from 'utils/i18n';
+
+
+const TopContributors = React.createClass({
+
+  propTypes: {
+    items: React.PropTypes.array.isRequired,
+    allItemsLoaded: React.PropTypes.bool.isRequired,
+    onLoadMore: React.PropTypes.func,
+  },
+
+  getInitialState() {
+    return {
+      items: this.props.items,
+      allItemsLoaded: this.props.allItemsLoaded,
+    };
+  },
+
+  loadMore() {
+    const _this = this;
+    this.props.onLoadMore((items, allItemsLoaded) => {
+      _this.setState({items, allItemsLoaded});
+    });
+  },
+
+  render() {
+    function createRow(item, index) {
+      const title = `<span class="value untranslated">${item.suggested}</span> suggested<br/>
+              <span class="value translated">${item.translated}</span> translated<br/>
+              <span class="value fuzzy">${item.reviewed}</span> reviewed<br/>`;
+      return (
+        <tr key={`top-contibutor-${index}`}>
+          <td className="number">{ t('# %(position)s', { position: index + 1})}</td>
+          <td className="user top-scorer">
+            <Avatar
+              email={item.email}
+              label={item.display_name}
+              size={20}
+              username={item.username}
+            />
+          </td>
+          <td className="number">
+            <span title={title}>
+              { item.public_total_score > 0 ?
```

Comment:
We should always try to keep the `render()` function simple and readable. In this case it makes sense to factor this out to a function:
```js
function getScoreText(score) {
  if (score > 0)
    return t('+%(score)s', { score });

  return score;
}
```
and use it as such:
```js
        <span title={title}>{getScoreText(item.public_total_score)</span>
```